assets/bpmn-js.css

```diff
@@ -1,8 +1,34 @@
 .bjs-container {
   --bjs-font-family: Arial, sans-serif;
-}
 
-.djs-container {
+  --color-grey-225-10-15: hsl(225, 10%, 15%);
```

Comment:
We do only need one color, correct? Why would we include all of themn?

Reply:
Correct, we only need certain colors (blue, black, white). I wanted to keep it consistent with diagram-js, where we also define colors we don't use (e.g. [`--color-grey-225-10-90`](https://github.com/bpmn-io/diagram-js/blob/develop/assets/diagram-js.css#L11)).

Reply:
In this case, O.K. I guess. At least we're consistent.